index.js

```diff
@@ -2632,6 +2642,7 @@
 //. [`Number`]:             #Number
 //. [`Object.create`]:      https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create
 //. [`Pair`]:               #Pair
+//. [`PositiveInteger`]:    #PositiveInteger
```

Comment:
I'm trying to cut down on the noise via `commit --amend` and `rebase`, but I fear I may be making things worse 😆